satellite/rewards/rewards.go

```diff
@@ -5,9 +5,28 @@ package rewards
 
 import (
 	"context"
+	"fmt"
+	"math"
+	"strconv"
 	"time"
 )
 
+// ToCents converts USD credit amounts to cents.
+func ToCents(dollar int) int {
+	cents := float64(dollar) * 100
+    return int(math.Round(cents))
+}
+
+// ToDollars converts credit amounts in cents to USD.
+func ToDollars(cents int) int {
+	formattedAmount := fmt.Sprintf("%d.%d",(cents / 100),(cents % 100))
+	dollars, err := strconv.ParseFloat(formattedAmount,64)
```

Comment:
Same as above

Reply:
Modulo and integer division has to be used per our group slack thread but results of those two have to be formatted and you can't format `%d.%d` as an int without truncating. Also this helps prevent needing to create a custom `Cent` type which I also realized would result in lots of breaking changes in the db logic.

Reply:
I think this is a display value. We just need to format it return the `formattedAmount`

Reply:
I'm going to write a new interface `Converter` to help invoke ToDollars as a method from the template without the need to change the type of `...CreditInCents`

Reply:
```golang
	s.templates.home, err = template.New("landingPage").Funcs(template.FuncMap{
		"ToDollars": func(cents int) string {
			return fmt.Sprintf("%d.%d", cents/100, cents%100)
		},
	}).ParseFiles(homeFiles...)
```
You can pass in a function like this as well and in the template you can do `{{ ToDollars .InviteeCreditInCents }}`

Reply:
this won't format properly with 505.

Reply:
If we only allow `integer` value for dollar amount. We should never have a cent value to be `505`?